lib/utils/format.js

```diff
@@ -419,6 +419,10 @@
         var className = /*'unipika-' + */node.$type.replace('.', '_');
         var title = '';
 
+        if (node.$category) {
+            className += WHITESPACE + 'pg_category_' + /*'unipika-' + */node.$category.toLowerCase()
```

Comment:
It would be useful to add some tests that will check resulting `className`.

Reply:
For sure.